fe/src/main/styl/bustracker.styl

```diff
@@ -0,0 +1,40 @@
+html,body 
+  height: 100%
+  width: 100%
+  margin: 0px
+  padding: 0px
+
+#container
+  position: absolute
+  top: 0
+  bottom: 0
+  left: 0
+  right: 0
+
+#mapcanvas 
+  width: 100%
+  height: 100%
+  position: absolute
+  top: 0
+  left: 0
+
+#info 
+  width: 100%
+  position: absolute
+  top: 0
+  left: 0
+  z-index: 10
+  opacity: 0.7
+  background-color: #EEE
```

Comment:
Most of the stuff above this point was already in the big file, although background-color was being set using JS rather than CSS